src/Cli.elm

```diff
@@ -191,7 +191,32 @@ generateFileFromOpenApiSpec config apiSpec =
                         )
                     |> BackendTask.map (\_ -> outputPath)
             )
-        |> BackendTask.andThen (\outputPath -> Pages.Script.log ("SDK generated at " ++ outputPath))
+        |> BackendTask.andThen
+            (\outputPath ->
+                let
+                    padLeft : { spaces : Int, text : String } -> String
+                    padLeft { spaces, text } =
+                        String.repeat spaces " " ++ text
+
+                    crlf : String
+                    crlf =
+                        "\u{000D}\n"
```

Comment:
Why not a simple `\n`?

Reply:
Windows terminals.

Reply:
I don't have a Windows system handy, does a plain `"\n"` not work in a Windows terminal?
If so, it's better to just use multiple calls to `Script.log`, combining them with `BackendTask.combine`

Reply:
I don't understand what the issue is:

> Text files created on DOS/Windows machines have different line endings than files created on Unix/Linux. DOS uses carriage return and line feed ("\r\n") as a line ending, which Unix uses just line feed ("\n").

This is a long-standing windows standard and `\n\n` wouldn't work in all terminals or environments in windows.

Reply:
I pushed a commit with what I think you are asking for but I don't really like it personally, maybe just a taste thing. 🤷🏻

Reply:
I think it's better because we probably don't want to output those `\r`s on Linux and macOS!
@wolfadex, what do you think?

Reply:
Linux and MacOS would ignore `\r` since they don't interpret it. Anyway, lets wait for @wolfadex to review the current version to get this half resolved. 😄

Reply:
They wouldn't ignore it? It would - e.g. - show up in log files and such?

Reply:
What does `Pages.Script.log` do when it comes to new lines? Does it do it per-system or does it always do the same thing regardless of system?

Reply:
Asking for my own knowledge too. Would be good to know for my `Ansi` package to make sure it handles things nicely since I tend to use it with `Pages.Script`

Reply:
`Pages.Script.log` calls `console.log`, which I expect to be system-specific

Reply:
https://nodejs.org/api/console.html#consolelogdata-args